include/grpc/impl/codegen/compression_types.h

```diff
@@ -58,9 +58,21 @@ typedef enum {
   GRPC_COMPRESS_DEFLATE,
   GRPC_COMPRESS_GZIP,
   /* TODO(ctiller): snappy */
+  GRPC_STREAM_COMPRESS_GZIP,
   GRPC_COMPRESS_ALGORITHMS_COUNT
 } grpc_compression_algorithm;
 
+#define GRPC_IS_STREAM_COMPRESSION_ALGORITHM(x) \
```

Comment:
all this becomes unnecessary if you create a separate enum for stream compression.